content/en/js3/blocks/async-await/index.md

```diff
@@ -0,0 +1,69 @@
++++
+title = '🍬 async/await'
+headless = true
+time = 20
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Define syntactic sugar'
+    2='Write a function using the async keyword'
++++
+
+```mermaid
+graph LR
+    Promise{{🤝 Promises}} --> |syntax| async{{🏃‍♂️ async}}
+    async --> |syntax| await{{📭 await}}
+    await --> |resolves to| Response{{📤 Response}}
+```
+
+Async/await is {{<tooltip title="syntactic sugar">}}A simpler, or "sweeter" way to write the same thing. The code works the same under the hood, but it's easier to read. {{</tooltip>}} for Promises. We group them together: async/await, because we {{<tooltip title="use them together. ">}}We can only use `await` inside an `async` function or at the [top level](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#top_level_await) of a module.{{</tooltip>}}
+
+We use the [`async`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) [keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#keywords) to define a function that returns a Promise. An async function always returns a Promise.
```

Comment:
For some reason the keywords anchor doesn't actually scroll to the right place in this page for me 🤷

Reply:
I KNOW IT IS SO ANNOYING